### ISSUE TITLE
Decide whether we really want all images to get screen reader text and, if so, align infrastructure

### ISSUE THREAD

**Issue by flannery-denny** [bootstrapworld/curriculum#799]
Currently our `@image` and `@centered-image` directives are very flexible. Wondering if that's actually a disservice to us because I keep finding mistakes and I know that Emmanuel wrote a commit not many weeks ago intended to guarantee that all images had text. 

We should discuss the possibility of establishing a single contract that all images have to follow... so that authors get error messages when they don't use it correctly.

Here's a concrete example of what I'm finding as result of allowing new and old contracts to work:

> @ifproglang{pyret}{@centered-image{images/gt-fun-pyret.png, 400, align="center"}}

- doubled up directions re: alignment
- image not sized as intended because the directive has changed since this was written
- screen reader text says "400" instead of a description

**Comment by schanzer** [comment]
@ds26gte I agree with @flannery-denny - _Every_ image should have alt text, and width can be optional. 

(Frankly I'd also like to get rid of `@centered-image{}` altogether, since it's not saving us much beyond `@span{.center}{@image{....}}`)

**Comment by flannery-denny** [comment]
@ds26gte @schanzer I think we should discuss this live rather than over git before implementing. I like @centered-image, but could retrain myself if need be.  

Flannery
